models/xception/xception.py

```diff
@@ -0,0 +1,63 @@
+from tensorflow.keras.layers import Activation, Add, BatchNormalization, Conv2D, Dense, GlobalAveragePooling2D, Input, MaxPool2D, SeparableConv2D
+from tensorflow.keras.models import Model
+
+# 299 x 299 x 3 images
+inputs = Input(shape=(299, 299, 3))
+
+# Entry flow
+outputs = Conv2D(filters=32, kernel_size=(3, 3), strides=(2, 2), use_bias=False)(inputs)
+outputs = BatchNormalization()(outputs)
+outputs = Activation("relu")(outputs)
+
+outputs = Conv2D(filters=64, kernel_size=(3, 3), use_bias=False)(outputs)
+outputs = BatchNormalization()(outputs)
+outputs = Activation("relu")(outputs)
+
+for filters in [128, 256, 728]:
```

Comment:
for filter in filters

Reply:
Renamed this variable to `num_filters` for better clarity.